File: JS, PHP, PYTHON/JAVASCRIPT/CICLO FOR.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

function inversion() {
    readline.question("Ingrese la cantidad a invertir: $", cantInput => {
        readline.question("Ingrese el interés anual: ", interInput => {
            readline.question("Ingrese el número de años: ", añosInput => {
                let cant = parseFloat(cantInput);
                let inter = parseFloat(interInput);
                let años = parseInt(añosInput, 10);

                for (let i = 0; i < años; i++) {
                    cant += cant * (inter / 100);
                    console.log(`En el año ${i + 1} tu capital obtenido será: $${cant.toFixed(2)}`);
                }
                readline.close();
            });
        });
    });
}

inversion();

File: JS, PHP, PYTHON/PYTHON/CICLO FOR.py
def inversion():
    cant = float(input("Ingrese la cantidad a invertir: $"))
    inter = float(input("Ingrese el interés anual: "))
    años = int(input("Ingrese el número de años: "))
    
    for i in range(años):
        cant = cant + (cant * (inter / 100))
        print(f"En un año tu capital obtenido sera: ${cant:.2f}")
        
inversion()
        